Lib/test/pickletester.py

```diff
@@ -2658,7 +2658,6 @@ def test_misc(self):
             self.assert_is_copy(x, y)
 
         # XXX test __reduce__ protocol?
-
```

Comment:
Also can keep this blank line?

Reply:
This was a typo on my part, have added back.